deployment/kubernetes/alerting/alertmanager/configmap.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alertmanager-config
+  namespace: kube-prometheus
+data:
+  alertmanager.yml: |
+    global:
+      resolve_timeout: 5m
+      smtp_from: 'CDN-Transcode-Sample@k8s.Prometheus'
+      smtp_smarthost: 'smtp.intel.com:25'
+      smtp_require_tls: false
+      smtp_hello: 'outlook.com'
+    route:
+      group_by: ['linux']
+      group_wait: 5s
+      group_interval: 5s
+      repeat_interval: 5m
+      receiver: 'email'
+    receivers:
+    - name: 'email'
+      email_configs:
+      - to: 'zhizhen.tang@intel.com'
```

Comment:
Do not use this particular email address.

Reply:
removed it.